CHANGELOG.md

```diff
@@ -1,10 +1,18 @@
-3.0.0 / TBD
+3.0.2 / TBD
+====================
+- Add `sessionToken` option to persist generated headers
+
```

Comment:
Should've been 3.1.0 since this is a feat, no?

Reply:
👍🏼